src/bitgo.js

```diff
@@ -184,6 +185,9 @@ const BitGo = function(params) {
     if (params.customSigningAddress) {
       common.Environments['custom'].customSigningAddress = params.customSigningAddress;
     }
+    if (params.microservicesUri) {
+      common.Environments['custom'].microservicesUri = params.microservicesUri;
```

Comment:
There is no existing code which makes use of the `microservicesUri` property on the environment (since this hasn't existed before). How come we store it in there, just to read it back out again later?

Reply:
when I asked you if there was a reason to store it there, you said a lot of places read from the common config for urls, so I thought adding it there may be useful to other places. but if you think it's unnecessary I'm fine with removing it.

Reply:
Yes, this is where we keep hardcoded values which we use across bitgojs. However I don't think this applies to your feature since 1) You aren't hard coding anything. 2) There are no other users of the microservices urls at this time.

Once we are comfortable exposing the microservices endpoint, we should add it as an entry in each env, but we don't need that at this point.

Reply:
I think Dan's thought is we don't put this in other environments, because dev/test/local/prd/whatever won't need the microservicesURl